java/src/jmri/jmrix/lenz/XNetTurnout.java

```diff
@@ -385,35 +383,23 @@ private synchronized void handleDirectModeFeedback(XNetReply l) {
             l.resetUnsolicited();
         }
         if (getCommandedState() != getKnownState() || internalState == COMMANDSENT) {
-            if (l.isFeedbackBroadcastMessage()) {
-                int numDataBytes = l.getElement(0) & 0x0f;
-                for (int i = 1; i < numDataBytes; i += 2) {
-                    int messageType = l.getFeedbackMessageType(i);
-                    if (messageType == 0 || messageType == 1) {
-                        if ((mNumber % 2 != 0
-                                && (l.getTurnoutMsgAddr(i) == mNumber))
-                                || (((mNumber % 2) == 0)
-                                && (l.getTurnoutMsgAddr(i) == mNumber - 1))) {
-                            // This message includes feedback for this turnout
-                            log.debug("Turnout {} DIRECT feedback mode - directed reply received.", mNumber);
-                            // set the reply as being solicited
-                            if (l.isUnsolicited()) {
-                                l.resetUnsolicited();
-                            }
-                            sendOffMessage();
```

Comment:
Removing this may be problematic.
The feedback has to be able to trigger an off message.
I will need to verify this works with real hardware.

Reply:
Please verify again. I am almost positive these lines were not removed, but **moved** to https://github.com/JMRI/JMRI/blob/980841c0e4b153293497e4e9d25444ffebafd953/java/src/jmri/jmrix/lenz/XNetTurnout.java#L396.

Similar for other supposed removals.

Reply:
OK.  It works with real hardware, which is what I care about.